crates/timers/src/future.rs

```diff
@@ -84,7 +84,7 @@ impl TimeoutFuture {
 ///
 /// # Example
 ///
-/// ```no_run
+/// ```compile_fail
```

Comment:
Why modify timers here? Let's do it in a separate PR

Reply:
This is purely to make CI happy or else CI wouldn't pass.

https://github.com/rustwasm/gloo/runs/4295589107?check_suite_focus=true